freetextresponse/tests.py

```diff
@@ -11,9 +11,10 @@
 
 from xblock.field_data import DictFieldData
 from xblock.validation import ValidationMessage
+from xblockutils.resources import ResourceLoader
 
 from django.db import IntegrityError
-from django.template.loader import get_template
+from django.template.context import Context
```

Comment:
Why the switch from `get_template()` to `Context()`?

Reply:
@caesar2164 I don't think it's switch as much as it's just a deletion of one import (`get_template`) and the addition of another (`Context`).

Reply:
@caesar2164 @stvstnfrd I've used `render_django_template` which is more XBlock friendly, especially for `i18n`.